packages/core/botpress/src/util.js

```diff
@@ -5,7 +5,7 @@ import fs from 'fs'
 import knex from 'knex'
 import generate from 'nanoid/generate'
 
-const IS_DEV = eval("process.env.NODE_ENV !== 'production'") /* eslint-disable-line no-eval */
+const IS_DEV = process.env.NODE_ENV !== 'production'
```

Comment:
@emirotin , this is because without `eval` `process.env.NODE_ENV` will be replaced to `production` during webpack-compilation of botpress-package. So if you are running bot in development with production build of webpack, it'd show `production`. See https://github.com/botpress/botpress/issues/591

Reply:
I see, thanks. Yes it shouldn't be an issue anymore, there's no webpack processing now so it will be what it is at runtime.